certificate_watcher.py

```diff
@@ -12,13 +12,14 @@
 from datetime import datetime, timedelta
 import socket
 import ssl
-import sys
 
 
 __version__ = "0.0.2"
 
+TLS_PORT = 443
```

Comment:
Maybe pedantic but I'd name it HTTPS_PORT.

Reply:
IMAPS is 993, and it's a TLS port.